package.json

```diff
@@ -27,5 +27,14 @@
 		"typescript": "latest",
 		"vite": "latest",
 		"vitest": "latest"
+	},
+	"pnpm": {
+		"overrides": {
+			"@vue/runtime-dom": "alpha",
+			"@vue/shared": "alpha",
+			"@vue/compiler-dom": "alpha",
+			"@vue/compiler-sfc": "alpha",
+			"vue": "alpha"
+		}
```

Comment:
Wouldn't doing overrides at this level also affect/break vue 2 tests?

Reply:
It does not affect Vue 2 tests. Since this PR will only be merged when Vue 3.5 is released, this json section will be removed before merging.